Gruntfile.js

```diff
@@ -176,6 +179,23 @@ module.exports = function (grunt) {
         grunt.file.write(manifestPath, JSON.stringify(manifest));
     });
 
+    grunt.registerTask('core', function () {
+        if (version) {
+            let done = this.async();
+            resolveRuntimeVersion(version).then(v => {
+                if(!grunt.file.exists(`${process.env.localAppData}/OpenFin/runtime/${v}`)){
```

Comment:
process.env.localAppData is undefined on Unix systems, this combined with failing if the runtime isn't found makes the tests impossible to run on Mac/Linux.